robottelo/ui/browser.py

```diff
@@ -42,12 +42,12 @@ def browser():
                 else webdriver.Chrome(
                     executable_path=settings.webdriver_binary)
             )
-        elif webdriver_name == 'ie':
+        elif webdriver_name == 'internet explorer':
             return (
                 webdriver.Ie() if settings.webdriver_binary is None
                 else webdriver.Ie(executable_path=settings.webdriver_binary)
             )
-        elif webdriver_name == 'edge':
+        elif webdriver_name == 'MicrosoftEdge':
```

Comment:
That's imo, but i think when we're talking about webdriver name, we should stick to webdriver's naming, not saucelab's :). In webdriver it's 'edge' (although 'internetexplorer' instead of 'ie', but not 'internet explorer').